funcx_web_service/routes/funcx.py

```diff
@@ -150,7 +151,15 @@ def auth_and_launch(user_id, function_uuid, endpoint_uuid, input_data, app, toke
     task = Task(rc, task_uuid, container_uuid, serializer, payload, task_group_id)
 
     task_channel.put(endpoint_uuid, task)
-    app.logger.info(f"Task:{task_uuid} placed on queue for endpoint:{endpoint_uuid}")
+
+    extra_logging = {
+        "user_id": user_id,
+        "task_id": task_uuid,
+        "task_group_id": task_group_id,
+        "endpoint_id": endpoint_uuid,
```

Comment:
Could we add function_id too?

Reply:
Yup

Reply:
Having `function_id` and `container_id` would be useful.